telegram/ext/_application.py

```diff
@@ -146,7 +146,7 @@ class Application(Generic[BT, CCT, UD, CD, BD, JQ]):
 
             .. seealso::
                 :meth:`add_handler`, :meth:`add_handlers`.
-        error_handlers (Dict[:obj:`callable`, :obj:`bool`]): A dict, where the keys are error
+        error_handlers (Dict[:term:`coroutine`, :obj:`bool`]): A dict, where the keys are error
```

Comment:
shouldn't it be `coroutine function`? [this](https://docs.python.org/3/glossary.html#term-coroutine) vs [this](https://docs.python.org/3/glossary.html#term-coroutine-function)